### ISSUE TITLE
Wrong prermissions for devices after icremental update from 3.2.2 to 3.3 

### ISSUE THREAD

**Issue by oysteinlunde** [docker/for-win#10910]
- [X ] I have tried with the latest version of Docker Desktop
  - [ X] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

After incremental update from 3.2.2 to 3.3 access to device files in `/dev/` has been changed for non-root users. The following files should have `rw` access for all users but has no access (`--`) after the 3.3 update.

```
crw-rw----  1 root root 245,   0 Apr  7 19:40 dxg
crw-rw----  1 root root   1,   7 Apr  7 02:46 full
crw-rw----  1 root root  10, 229 Apr  7 02:46 fuse
crw-rw----  1 root root   1,   3 Apr  7 02:46 null
crw-rw----  1 root root   5,   2 Apr 12 17:27 ptmx
crw-rw----  1 root root   1,   8 Apr  7 02:46 random
crw-rw----  1 root root   5,   0 Apr  7 09:39 tty
crw-rw----  1 root root   1,   9 Apr  7 02:46 urandom
crw-rw----  1 root root   1,   5 Apr  7 02:46 zero
```

Running the full installasjon of 3.3 from https://docs.docker.com/docker-for-windows/release-notes/ corrects the problem.

### Actual behavior

`crw-rw---- 1 root root 1, 3 Apr 13 07:23 /dev/null`

### Expected behavior

`crw-rw-rw- 1 root root 1, 3 Apr 13 07:23 /dev/null`

### Information

  - Windows Version: Win10EP-1909 (18363.1440)
  - Docker Desktop Version:  3.3 (incremental update)
  - WSL2 or Hyper-V backend?  Hyper-V
  - Are you running inside a virtualized Windows e.g. on a cloud server or a VM: Physical

### Steps to reproduce the behavior

  1. `docker run -ti ubuntu:focal sh -c 'ls -l /dev/null'`

**Comment by stephen-turner** [comment]
Thanks for the report. I suspect this is the same as docker/for-mac#5527, which we are working on a fix for, but we'll check.

**Comment by oysteinlunde** [comment]
Fixed in Docker Desktop release 3.3.1
